objectivec/Tests/unittest_objc_options.proto

```diff
@@ -0,0 +1,61 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Done forget to add this to the Makefile.am so it is known to the outer build system